nfc/include/nfc_platform.h

```diff
@@ -111,6 +122,31 @@ void nfc_platform_cb_request(const void    * p_ctx,
                              size_t          data_len,
                              bool            copy_data);
 
+/** @brief Allocate buffer for NFC data exchange.
+ *
+ * This function allocates a buffer that is used directly by the NFCT peripheral.
+ * You might need to allocate buffer within defined memory addresses that are accessible
+ * by the EasyDMA NFCT peripheral utility. Check your Product Specification for more details.
+ *
+ * The buffer size varies for different NFC tag types:
+ *
+ *   - NFC Type 2 Tag: 16 bytes @ref NFC_PLATFORM_T2T_BUFFER_SIZE
+ *   - NFC Type 4 Tag: 259 bytes @ref NFC_PLATFORM_T4T_BUFFER_SIZE
+ *
+ * @param[in] size Size of the requested buffer in bytes.
+ *
+ * @retval Address of allocated buffer if successful, otherwise NULL.
+ */
+uint8_t * nfc_platform_buffer_alloc(size_t size);
+
+/** @brief Free allocated buffer.
+ *
+ * If @a p_buffer is NULL, no operation is performed.
```

Comment:
What is that @a?

Reply:
As I know the @a renders the next word in italics. It's supposed to be used to reference parameters.

Reply:
Roger that.